RevenueCatUI/Templates/Components/TemplateComponentsView.swift

```diff
@@ -4,19 +4,33 @@
 //
 //  Created by Josh Holtz on 6/11/24.
 //
-// swiftlint:disable missing_docs todo
+// swiftlint:disable missing_docs
 
 import RevenueCat
 import SwiftUI
 
 #if PAYWALL_COMPONENTS
+
+class PaywallState: ObservableObject {
+
+    @Published var selectedPackage: Package?
+
+    func select(package: Package) {
+        self.selectedPackage = package
+    }
+
+}
+
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
-public struct TemplateComponentsView: View {
+struct TemplateComponentsView: View {
 
     let paywallComponentsData: PaywallComponentsData
     let componentViewModels: [PaywallComponentViewModel]
     private let onDismiss: () -> Void
 
+    @StateObject
+    private var paywwallState = PaywallState()
```

Comment:
Typo
```suggestion
    private var paywallState = PaywallState()
```

Reply:
I need you all to start letting me have a few of these typos... I try so hard to get them merged in 🤦‍♂️